homeassistant/components/ozw/__init__.py

```diff
@@ -101,7 +101,8 @@ def async_instance_event(message):
         _LOGGER.debug("[INSTANCE EVENT]: %s - data: %s", event, event_data)
         # The actual removal action of a Z-Wave node is reported as instance event
         # Only when this event is detected we cleanup the device and entities from hass
-        if event == "removenode" and "Node" in event_data:
+        # TODO: Find a more elegant way of doing this, e.g. a notification of this event from OZW
```

Comment:
This might fail pylint. Probably remove TODO.

Reply:
Last time I checked it isn't set that strict. It warns on my machine but doesn't fail. I actually like the warning so I'm remembered ;-)

But offcourse if it fails, I'll adjust.

Reply:
You were right, pylint was not happy with the TODO comment. Fixed.